extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/TransferHistoryEntryContractAgreementDetails.java

```diff
@@ -0,0 +1,32 @@
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+import java.time.OffsetDateTime;
+
+@Data
+public class TransferHistoryEntryContractAgreementDetails {
```

Comment:
these fields can be contained in the parent object, the nesting is not necessarily required.